pyproject.toml

```diff
@@ -1,3 +1,79 @@
+[project]
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
+name = "phalanx"
+version = "1.0.0"
+description = "Python support code for the Rubin Phalanx platform."
+license = {file = "LICENSE"}
+readme= "README.rst"
+keywords = [
+    "rubin",
+    "lsst",
+]
+# https://pypi.org/classifiers/
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Intended Audience :: Developers",
+    "Operating System :: POSIX",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "GitPython",
+]
```

Comment:
I think this should only be a dev dependency since it's only used by a test script and is not necessary to build the docs or install Phalanx, but this is probably just an aspect of my other comment.